src/test/kotlin/cacophony/regex/RegexParserTest.kt

```diff
@@ -9,6 +9,58 @@ import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.assertThrows
 
 class RegexParserTest {
+    private fun algebraicRegexEquals(
+        x: AlgebraicRegex,
+        y: AlgebraicRegex,
```

Comment:
Please add at least some comments that it doesn't cover some situation, like (a|b) (b|a)

Reply:
Done